packages/rest/src/rest-server.ts

```diff
@@ -524,10 +546,15 @@ export class RestServer extends Context implements Server {
   async start(): Promise<void> {
     // Setup the HTTP handler so that we can verify the configuration
     // of API spec, controllers and routes at startup time.
-    this._setupHandlerIfNeeded();
-
     const httpPort = await this.get(RestBindings.PORT);
     const httpHost = await this.get(RestBindings.HOST);
+    // TBD: add integration test for it.
+    // get api after start
+    // Verify the explorer endpoints work
+    this._addServerSpec({hostname: httpHost, port: httpPort});
```

Comment:
We should NOT be adding explicit host/port to OpenAPI spec defined by the application developer, see my previous comment for explanation.

Let's keep the current behavior please, there are important reasons why the current Swagger-based implementation is not automatically filling host/port values in the developer-provided spec.